### PR TITLE
Ref #task 7046 Contact

### PR DESCRIPTION
**Pull request by hieungv**
## Redmine URL
 https://edu-redmine.framgia.vn/issues/7046

## Reproduce steps
 - for Bug only.

## What kind of page or function is need to check
 - 
 - 
 - 

## Other
image
image

image

### CODE REVIEW

**Comment by hieungv** [comment]
ready

File: app/models/feedbackss.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Feedbackss < ApplicationRecord
```
Comment (yanghoxom) [inline app/models/feedbackss.rb]: K validate à, feedback phải có body chứ vs info chư

File: app/controllers/feedbacks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  def edit; end
+
+  def create
+    @feedback = Feedback.new(feedback_params)
+
+    respond_to do |format|
+      if @feedback.save
+        format.html do
+          redirect_to @feedback, notice: "Feedback was successfully created."
```
Comment (yanghoxom) [inline app/controllers/feedbacks_controller.rb]: I18n

File: app/controllers/feedbacks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        end
+        format.json{render :show, status: :created, location: @feedback}
+      else
+        format.html{render :new}
+        format.json do
+          render json: @feedback.errors, status: :unprocessable_entity
```
Comment (yanghoxom) [inline app/controllers/feedbacks_controller.rb]: Có cả xử lý ajax à mà xài json

File: app/views/feedbacks/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      <h2><%= pluralize(@feedback.errors.count, "error") %> prohibited this feedback from being saved:</h2>
+
+      <ul>
+      <% feedback.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+    <h4>Email Us</h4>
```
Comment (yanghoxom) [inline app/views/feedbacks/_form.html.erb]: I18n

**Comment by hieungv** [comment]
ready

**Review by yanghoxom — COMMENTED** [review]
Kiểm tra i18n nhé

**Comment by hieungv** [comment]
ready

**Pull request merged**